### PR TITLE
adds rss feed of latest RFC threads

### PR DESCRIPTION
**Pull request by agallou**
screen shot 2018-09-01 at 19 15 16

### CODE REVIEW

File: res/views/layout.html.twig [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="alternate" type="application/rss+xml" href="/rss" />
```
[the remark this answers is not in the document]
Comment (agallou) [reply res/views/layout.html.twig]: should the title be "Latest emails" ?
Right now it seems to be a bug, the feed displays the first 100 emails : https://github.com/mnapoli/externals/blob/e8988e3d51c33765fe134a48b8029b7a15901a2b/src/Email/EmailRepository.php#L175
Is that the expected behaviour ?

File: res/http.php [hunk 2]
```diff
@@ -109,7 +110,12 @@
             $emails = $emailRepository->findLatest($since);
             return $rss->build($emails);
         },
-
+        '/rss-rfc' => function (EmailRepository $emailRepository, ServerRequestInterface $request) {
+            newrelic_name_transaction('rss-rfc');
+            $rss = new RssRfcBuilder($request);
+            $threads = $emailRepository->findThreads("where threadInfos.subject like '%RFC%'", 'order by threadInfos.date desc', 1, null);
```
Comment (mnapoli) [inline res/http.php]: Oh I think the `findThreads()` method is public by mistake (only called in `EmailRepository`). Could you make it private and add a new method like `$emailRepository->findRfcThreads()` (like `findTopThreads()` for example)?

File: res/views/layout.html.twig (hunk 1 above)
[the remark this answers is not in the document]
Comment (mnapoli) [reply res/views/layout.html.twig]: "Latest emails" -> 👍 

> Right now it seems to be a bug, the feed displays the first 100 emails

I don't understand what the problem is? Is that supposed to display more? Less?

[the remark this answers is not in the document]
Comment (staabm) [reply res/views/layout.html.twig]: I guess he is refering to the ID ASCENDING thing, which sorts the oldest not the latest stuff

[the remark this answers is not in the document]
Comment (agallou) [reply res/views/layout.html.twig]: yes, that's it. we display only the first 100 emails and not en last 100 mails.
If your ok with it after this PR I could do another one to chnage the order and add the title.

File: res/http.php (hunk 2 above)
Comment (agallou) [reply res/http.php]: I've moved the method in the repository and made the other one private.

File: res/views/layout.html.twig (hunk 1 above)
[the remark this answers is not in the document]
Comment (mnapoli) [reply res/views/layout.html.twig]: :+1:

**Comment by mnapoli** [comment]
Thanks!

**Pull request merged**